src/test/java/com/google/jenkins/plugins/computeengine/ComputeEngineCloudWindowsIT.java

```diff
@@ -0,0 +1,299 @@
+package com.google.jenkins.plugins.computeengine;
+
+import com.cloudbees.jenkins.plugins.sshcredentials.impl.BasicSSHUserPrivateKey;
+import com.cloudbees.plugins.credentials.*;
+import com.cloudbees.plugins.credentials.common.StandardUsernameCredentials;
+import com.cloudbees.plugins.credentials.domains.Domain;
+import com.cloudbees.plugins.credentials.domains.DomainRequirement;
+import com.google.api.services.compute.model.Image;
+import com.google.api.services.compute.model.Instance;
+import com.google.api.services.compute.model.Operation;
+import com.google.jenkins.plugins.computeengine.client.ClientFactory;
+import com.google.jenkins.plugins.computeengine.client.ComputeClient;
+import com.google.jenkins.plugins.computeengine.ssh.GoogleKeyPair;
+import com.google.jenkins.plugins.credentials.oauth.GoogleRobotPrivateKeyCredentials;
+import com.google.jenkins.plugins.credentials.oauth.ServiceAccountConfig;
+import hudson.model.Node;
+import hudson.model.labels.LabelAtom;
+import hudson.slaves.NodeProvisioner;
+import org.junit.*;
```

Comment:
Replace * imports with discrete imports.

Reply:
done. Think IntelliJ automatically swapped it.